db/models.py

```diff
@@ -1 +1,29 @@
+from typing import ForwardRef
+
 from django.db import models
+from django.db.models import SET_NULL, CASCADE
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField()
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True, )
+    description = models.TextField(null=True)
```

Comment:
Ensure consistency in the `description` field's nullability. It is set to `null=True` here, which should match the migration settings.